sql/core/src/test/resources/sql-tests/inputs/udf/udf-inline-table.sql

```diff
@@ -0,0 +1,53 @@
+-- This test file was converted from inline-table.sql.
+-- [SPARK-28291] UDFs cannot be evaluated within inline table definition
```

Comment:
@imback82, can we add a todo that says we should add UDFs in VALUES clause when SPARK-28291 is resolved? It's kind of sad because I actually intended to test such cases here.

Reply:
e.g.:

```
-- TODO: we should ...
```

Reply:
Or .. you can make one test failed and comment it out with linking the JIRA. e.g. https://github.com/apache/spark/blob/master/sql/core/src/test/resources/sql-tests/inputs/udf/pgSQL/udf-aggregates_part1.sql#L65-L67